Generate_tile.scad

```diff
@@ -38,27 +41,27 @@ module letter(l) {
 }
 
 
-text_height = (tile_length - letter_size);
-text_centre = (tile_width / 2);
+text_height = (final_length - letter_size);
+text_centre = (final_width / 2);
 
-translate([text_height,text_centre,0]) rotate([0,0,90]) letter(text);
```

Comment:
Using 0 as starting plane for drawing the text and the icon (so it was inside the tile body). Because of this you always had to set letter_height with a value greater than the tile height. This also was causing some weird behaviour while printing the mid layers (at some sizes the printer was trying to draw the icon partially in the mid layers making it slower) I updated lines 44 and 98 so the drawing sits on top of the tile.